README.md

```diff
@@ -1,1193 +1,25 @@
-# Ballerina Azure Event Hubs Connector
+Ballerina Azure Event Hubs Connector
+===================
 
 [![Build](https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/actions?query=workflow%3ACI)
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-azure.eventhub.svg)](https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/commits/master)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Connects to Microsoft Azure Event Hubs using Ballerina.
-
-# Introduction
-
-## What is Azure Event Hub?
-
 [Azure Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-about) is a fully managed, real-time 
 data ingress service which is highly scalable, secured, open and reliable that ingests data (events) from a variety of 
 sources and reliably distribute between multiple independent 
-systems for processing, storage and analysis. Azure Event Hubs streamline the data pipelines so that the journey from 
-inputs to insights is seamless.  It makes sure all the incoming data is received reliably, fast, in order and is 
-available for consumption by many processing systems at their own pace. It automatically decouples event producers from 
-event consumers untangling the system architecture. With the elastic scale, event hubs have the durability to handle 
-massive amounts of data across various types, sizes and speeds. Event hub integrates with other azure services 
-to unlock insights. 
-
-## Key Features of Azure Event Hub
+systems for processing, storage and analysis.
 
-* Ingest millions of events per second from hundreds of thousands of sources with low latency and configurable 
-  time retention.
-* Enable real time and micro batch processing concurrently.
-* Handle massive amounts of data across various types, sizes and speeds with elastic scale.
-* Easily connect with Apache Kafka ecosystem
-* Natively connect with [Stream Analytics](https://azure.microsoft.com/en-us/services/stream-analytics/) to build an 
-end-to-end serverless streaming solution.
-
-![image](docs/images/Azure_Event_Hubs.png)
-
-## Connector Overview
-
-Azure Event Hubs Ballerina Connector is used to connect with the Azure Event Hubs to ingest millions of events per 
+Azure Event Hubs [Ballerina](https://ballerina.io/) Connector is used to connect with the Azure Event Hubs to ingest millions of events per 
```

Comment:
Connector -> connector

Reply:
Resolved and amended the changes. Thank you for the suggestion.